spaces/S000181/README.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: S000181
+name: Countable sigma product on ω_1+1
```

Comment:
Would it be possible to use Latex for $\omega_1+1$?  I tried to type "omega" in the search box for spaces and without this, the space is not picked up.

Reply:
Yes